### PR TITLE
Initial structure and setup for development

### PR DESCRIPTION
**Pull request by subashcs**
This Pull Request contains the following:

-  package picker , the actual library we are implementing
- test and lint setup 
-  package example, the example website for test and documentation hosting
-  lerna wrapper for managing mono repo
-  ci setup for package publishing currently on dry run

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (sugat009) [inline .vscode/launch.json:1]: Are these vscode specific files really necessary to be committed?

File: packages/picker/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Changelog
+
+Full release notes at https://github.com/subashcs/releases
```
Comment (sugat009) [inline packages/picker/CHANGELOG.md:3]: Link typo?

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+
+## Main features
+- ♿️ [WAI-ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) support
+- 🤖 Written in TypeScript
+
+## Installation
+
+```
+yarn add subash-date-picker-react
```
Comment (sugat009) [inline README.md:13]: Typo

File: lerna.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (sugat009) [inline lerna.json:1]: Do we need to be working with a monorepo structure?

**Review by sugat009 — CHANGES_REQUESTED** [review]

File: lerna.json (hunk 4 above)
Comment (aj3sh) [reply lerna.json:1]: If `learna` isn't helping us then let's remove it.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  release:
+    types: [created]
```
Comment (aj3sh) [inline .github/workflows/ci.yml:12]: It's better if we maintain the release job on a separate GitHub action. Like `release.yml`

File: CONTRIBUTING.MD [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+8. Verify your test cases and update examples on the example web app. 
+9. Commit changes and create a Pull Request. 
+10. Once the PR is merged to master, this will publish the library.
+
+
+## Running Unit Tests
+We are using Jest to Run unit tests.
+
+## Configuring VSCode development environment
+We have configured basic workspace configuration and automated tasks that are run when opening our workspace on vscode
```
Comment (aj3sh) [inline CONTRIBUTING.MD:21]: The vscode configurations are mentioned here. That's why they are added to the repository. What do you think? @sugat009

File: packages/example/package.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    "start": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "subashcs-react-date-picker": "0.0.1",
```
Comment (aj3sh) [inline packages/example/package.json:16]: Does this package really depends on `subashcs-react-date-picker` ??

**Review by aj3sh — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.MD (hunk 6 above)
Comment (sugat009) [reply CONTRIBUTING.MD:21]: it's a nice feature to have but not everyone might be using vscode
let's put the configs in a separate repo, dedicated to configs, where users can pull from there

**Pull request closed without merging**